docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -415,6 +415,24 @@ the entire substring is returned by default.
 For example, `REGEXP_EXTRACT("(.*) (.*)", 'hello there', 2)`
 returns "there".
 
+### REGEXP_EXTRACT_ALL
+
+```sql
+REGEXP_EXTRACT_ALL('.*', col1)
+```
+
+```sql
+REGEXP_EXTRACT_ALL('(([AEIOU]).)', col1, 2)
+```
+
+Extract the all subtrings matched by the regex pattern from the input.
```

Comment:
Is this correct english? :P  "the all"? You know better, just wondering

Reply:
Oops, extra the.  Good catch